test/integration/agent/cniserver_test.go

```diff
@@ -740,10 +754,21 @@ func setupChainTest(
 		routeMock = routetest.NewMockInterface(controller)
 		podNetworkWait := wait.NewGroup()
 		flowRestoreCompleteWait := wait.NewGroup()
+		kubeClient := k8sFake.NewSimpleClientset()
+		localPodInformer := coreinformers.NewFilteredPodInformer(
+			kubeClient,
+			metav1.NamespaceAll,
+			0,
+			cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc},
+			func(options *metav1.ListOptions) {
+				options.FieldSelector = fields.OneTermEqualSelector("spec.nodeName", "fakeNode").String()
+			},
+		)
```

Comment:
maybe add a helper function to create the informer, to avoid the duplication

for consistency, you may want to use the same fake Node Name as in the `testNodeConfig`

Reply:
After moving the podInformer/podLister fields to podConfigurator, we don't need to initialize localPodInformer on Linux, removed the change in this file.